thunderbird/MozillaThunderbird.spec

```diff
@@ -306,9 +307,6 @@ export MOZ_BUILD_DATE=%{releasedate}
 export MOZILLA_OFFICIAL=1
 export BUILD_OFFICIAL=1
 export MOZ_TELEMETRY_REPORTING=1
-%if %{update_channel} == "esr"
-export MOZ_ESR=1
-%endif
```

Comment:
Thunderbird does not have a real ESR release since years.

Reply:
Good. Less code is always better.